pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -25,6 +32,5 @@ stdenv.mkDerivation rec {
     description = "Build tool that builds code quickly and reliably";
     license = stdenv.lib.licenses.asl20;
     maintainers = [ stdenv.lib.maintainers.philandstuff ];
-    platforms = [ "x86_64-linux" ];
```

Comment:
build failure on darwin:

```
$ nix-build -A bazel
these derivations will be built:
  /nix/store/4w8d6phb81996qgkph1h0hv7izif72zc-zulu1.8.0_66-8.11.0.1.drv
  /nix/store/r407rs601jvrz9ib7x55mi6cwkdyx0sv-bazel-0.3.1.drv
these paths will be fetched (71.33 MiB download, 74.94 MiB unpacked):
  /nix/store/c9i30rqzlixj4sn0hbhd3255rlqb7jzq-zulu1.8.0_66-8.11.0.1-macosx.zip
fetching path ‘/nix/store/c9i30rqzlixj4sn0hbhd3255rlqb7jzq-zulu1.8.0_66-8.11.0.1-macosx.zip’...

*** Downloading ‘http://cache.nixos.org/nar/040nm324yaf4bd0asyfyhkca7k38m45xqbycy9y9f4dyxkdvn5ik.nar.xz’ to ‘/nix/store/c9i30rqzlixj4sn0hbhd3255rlqb7jzq-zulu1.8.0_66-8.11.0.1-macosx.zip’...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 71.3M  100 71.3M    0     0  21.7M      0  0:00:03  0:00:03 --:--:-- 21.7M

building path(s) ‘/nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1’
unpacking sources
unpacking source archive /nix/store/c9i30rqzlixj4sn0hbhd3255rlqb7jzq-zulu1.8.0_66-8.11.0.1-macosx.zip
source root is zulu1.8.0_66-8.11.0.1-macosx
setting SOURCE_DATE_EPOCH to timestamp 1445964344 of file zulu1.8.0_66-8.11.0.1-macosx/jre/lib/libfontmanager.dylib
patching sources
configuring
no configure script, doing nothing
building
no Makefile, doing nothing
installing
post-installation fixup
moving /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/man to /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/share/man
gzipping man pages in /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1
stripping (with flags -S) in /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib  /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/bin
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/ct.sym
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/dt.jar
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/ir.idl
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/jconsole.jar
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/orb.idl
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/sa-jdi.jar
strip: can't process non-object and non-archive file: /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/lib/tools.jar
patching script interpreter paths in /nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1
building path(s) ‘/nix/store/chsnz2nrz9fdmlslv2mcd316xgrby5dv-bazel-0.3.1’
unpacking sources
unpacking source archive /nix/store/aiq6xr4ydrkbndylqc9m8df2bz60pkkz-bazel-0.3.1-src
source root is bazel-0.3.1-src
patching sources
configuring
no configure script, doing nothing
building
INFO: You can skip this first step by providing a path to the bazel binary as second argument:
INFO:    compile.sh compile /path/to/bazel
🍃  Building Bazel from scratch......which: no git in (/nix/store/si5xksicwx3r5qm0b257im4x7glyib2d-pkg-config-0.29/bin:/nix/store/40hxqmrcv6zp3h192c6pqj53mx36d4bi-protobuf-3.0.0/bin:/nix/store/gfyqib807w47g25z4qibh7yf4b018mzk-zip-3.0/bin:/nix/store/v8zl8282jgz9b2pqcmy8ldhqqzl78i4r-libarchive-3.2.1/bin:/nix/store/h9zswwypdln90nasrgzl3rwrxyv1s06j-unzip-6.0/bin:/nix/store/9rxxfn5bc79bc8yqcs6p4rkrz35wsy11-which-2.21/bin:/nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/bin:/nix/store/ca855d33qlz2l4872p8y9fhhrzknjbm2-clang-wrapper-3.7.1/bin:/nix/store/v3c98r884y4g3ff276xc98gvxan5dpbv-clang-3.7.1/bin:/nix/store/b5ql7i4yfwhkgfa5nkfydsc5s73g50lw-cctools-binutils-darwin/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/3bqvg4z7a0j8ynkwbyblni3rwpcwyr29-findutils-4.6.0/bin:/nix/store/23savmq8d7hsv51glxqqnjahrm428j07-diffutils-3.5/bin:/nix/store/98amxzf6xnidvi7wczkcc9wacqck5x51-gnused-4.2.2/bin:/nix/store/jsp5i5shg6j59a7f11yn87szakqn2sbp-gnugrep-2.25/bin:/nix/store/hncnvp5mpayp0db4ryn8m1yy6yy7xz49-gawk-4.1.3/bin:/nix/store/y7z75inj0a1yf70a40ywfsjpym4k2ays-gnutar-1.29/bin:/nix/store/qh0xv89nlaidc1s6scr9r89vdc1ba6dp-gzip-1.8/bin:/nix/store/zcc294drrgp3iphpwlpkpxz684117yfc-bzip2-1.0.6.0.1-bin/bin:/nix/store/mygp9fxpdg4qn6pl0xhr4yddvxd2iifk-gnumake-4.2.1/bin:/nix/store/w1k6yj06bfcsmyfvc9iki36dlczqscx5-bash-4.3-p46/bin:/nix/store/25byczn2dp79h5vcr96c5znhrggwb7fi-patch-2.7.5/bin:/nix/store/4nwk5kr428ds9bj5przdbrs8mqmllxxg-xz-5.2.2-bin/bin)
which: no git in (/nix/store/si5xksicwx3r5qm0b257im4x7glyib2d-pkg-config-0.29/bin:/nix/store/40hxqmrcv6zp3h192c6pqj53mx36d4bi-protobuf-3.0.0/bin:/nix/store/gfyqib807w47g25z4qibh7yf4b018mzk-zip-3.0/bin:/nix/store/v8zl8282jgz9b2pqcmy8ldhqqzl78i4r-libarchive-3.2.1/bin:/nix/store/h9zswwypdln90nasrgzl3rwrxyv1s06j-unzip-6.0/bin:/nix/store/9rxxfn5bc79bc8yqcs6p4rkrz35wsy11-which-2.21/bin:/nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/bin:/nix/store/ca855d33qlz2l4872p8y9fhhrzknjbm2-clang-wrapper-3.7.1/bin:/nix/store/v3c98r884y4g3ff276xc98gvxan5dpbv-clang-3.7.1/bin:/nix/store/b5ql7i4yfwhkgfa5nkfydsc5s73g50lw-cctools-binutils-darwin/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/3bqvg4z7a0j8ynkwbyblni3rwpcwyr29-findutils-4.6.0/bin:/nix/store/23savmq8d7hsv51glxqqnjahrm428j07-diffutils-3.5/bin:/nix/store/98amxzf6xnidvi7wczkcc9wacqck5x51-gnused-4.2.2/bin:/nix/store/jsp5i5shg6j59a7f11yn87szakqn2sbp-gnugrep-2.25/bin:/nix/store/hncnvp5mpayp0db4ryn8m1yy6yy7xz49-gawk-4.1.3/bin:/nix/store/y7z75inj0a1yf70a40ywfsjpym4k2ays-gnutar-1.29/bin:/nix/store/qh0xv89nlaidc1s6scr9r89vdc1ba6dp-gzip-1.8/bin:/nix/store/zcc294drrgp3iphpwlpkpxz684117yfc-bzip2-1.0.6.0.1-bin/bin:/nix/store/mygp9fxpdg4qn6pl0xhr4yddvxd2iifk-gnumake-4.2.1/bin:/nix/store/w1k6yj06bfcsmyfvc9iki36dlczqscx5-bash-4.3-p46/bin:/nix/store/25byczn2dp79h5vcr96c5znhrggwb7fi-patch-2.7.5/bin:/nix/store/4nwk5kr428ds9bj5przdbrs8mqmllxxg-xz-5.2.2-bin/bin)

🍃  Building Bazel with Bazel.
.WARNING: /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel_iCe56OYp/out/external/bazel_tools/WORKSPACE:1: Workspace name in /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel_iCe56OYp/out/external/bazel_tools/WORKSPACE (@io_bazel) does not match the name given in the repository's definition (@bazel_tools); this will cause a build error in future versions.
WARNING: /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel-0.3.1-src/src/main/native/BUILD:52:16: in includes attribute of cc_binary rule //src/main/native:libunix.so: '.' resolves to 'src/main/native' not in 'third_party'. This will be an error in the future.
WARNING: /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel-0.3.1-src/src/main/protobuf/BUILD:36:1: in includes attribute of cc_library rule //src/main/protobuf:command_server_cc_proto: '.' resolves to 'src/main/protobuf' not in 'third_party'. This will be an error in the future. Since this rule was created by the macro 'cc_grpc_library', the error might have been caused by the macro implementation in /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel-0.3.1-src/tools/build_rules/genproto.bzl:104:18.
INFO: Found 1 target...
INFO: From Compiling src/main/tools/dummy-sandbox.c:
In file included from <built-in>:328:
<command line>:6:9: warning: '_FORTIFY_SOURCE' macro redefined [-Wmacro-redefined]
#define _FORTIFY_SOURCE 2
        ^
<command line>:2:9: note: previous definition is here
#define _FORTIFY_SOURCE 1
        ^
1 warning generated.
ERROR: /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel-0.3.1-src/src/main/cpp/util/BUILD:24:1: C++ compilation of rule '//src/main/cpp/util:md5' failed: cc_wrapper.sh failed: error executing command
  (cd /private/tmp/nix-build-bazel-0.3.1.drv-0/bazel_iCe56OYp/out/execroot/bazel-0.3.1-src && \
  exec env - \
    LD_LIBRARY_PATH=/nix/store/v3c98r884y4g3ff276xc98gvxan5dpbv-clang-3.7.1/lib \
    PATH=/nix/store/si5xksicwx3r5qm0b257im4x7glyib2d-pkg-config-0.29/bin:/nix/store/40hxqmrcv6zp3h192c6pqj53mx36d4bi-protobuf-3.0.0/bin:/nix/store/gfyqib807w47g25z4qibh7yf4b018mzk-zip-3.0/bin:/nix/store/v8zl8282jgz9b2pqcmy8ldhqqzl78i4r-libarchive-3.2.1/bin:/nix/store/h9zswwypdln90nasrgzl3rwrxyv1s06j-unzip-6.0/bin:/nix/store/9rxxfn5bc79bc8yqcs6p4rkrz35wsy11-which-2.21/bin:/nix/store/s2h7gqw4fz1lb92vcpk03hhm02dv1p50-zulu1.8.0_66-8.11.0.1/bin:/nix/store/ca855d33qlz2l4872p8y9fhhrzknjbm2-clang-wrapper-3.7.1/bin:/nix/store/v3c98r884y4g3ff276xc98gvxan5dpbv-clang-3.7.1/bin:/nix/store/b5ql7i4yfwhkgfa5nkfydsc5s73g50lw-cctools-binutils-darwin/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/5q72gfbqjw1mblviazmydgr8zjz9fz4a-coreutils-8.25/bin:/nix/store/3bqvg4z7a0j8ynkwbyblni3rwpcwyr29-findutils-4.6.0/bin:/nix/store/23savmq8d7hsv51glxqqnjahrm428j07-diffutils-3.5/bin:/nix/store/98amxzf6xnidvi7wczkcc9wacqck5x51-gnused-4.2.2/bin:/nix/store/jsp5i5shg6j59a7f11yn87szakqn2sbp-gnugrep-2.25/bin:/nix/store/hncnvp5mpayp0db4ryn8m1yy6yy7xz49-gawk-4.1.3/bin:/nix/store/y7z75inj0a1yf70a40ywfsjpym4k2ays-gnutar-1.29/bin:/nix/store/qh0xv89nlaidc1s6scr9r89vdc1ba6dp-gzip-1.8/bin:/nix/store/zcc294drrgp3iphpwlpkpxz684117yfc-bzip2-1.0.6.0.1-bin/bin:/nix/store/mygp9fxpdg4qn6pl0xhr4yddvxd2iifk-gnumake-4.2.1/bin:/nix/store/w1k6yj06bfcsmyfvc9iki36dlczqscx5-bash-4.3-p46/bin:/nix/store/25byczn2dp79h5vcr96c5znhrggwb7fi-patch-2.7.5/bin:/nix/store/4nwk5kr428ds9bj5przdbrs8mqmllxxg-xz-5.2.2-bin/bin \
    TMPDIR=/private/tmp/nix-build-bazel-0.3.1.drv-0 \
  external/local_config_cc/cc_wrapper.sh -U_FORTIFY_SOURCE '-D_FORTIFY_SOURCE=1' -fstack-protector -Wall -Wthread-safety -Wself-assign -fcolor-diagnostics -fno-omit-frame-pointer '-std=c++0x' -MD -MF bazel-out/local-fastbuild/bin/src/main/cpp/util/_objs/md5/src/main/cpp/util/md5.pic.d '-frandom-seed=bazel-out/local-fastbuild/bin/src/main/cpp/util/_objs/md5/src/main/cpp/util/md5.pic.o' -fPIC -iquote . -iquote bazel-out/local-fastbuild/genfiles -iquote external/bazel_tools -iquote bazel-out/local-fastbuild/genfiles/external/bazel_tools -isystem external/bazel_tools/tools/cpp/gcc3 -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c src/main/cpp/util/md5.cc -o bazel-out/local-fastbuild/bin/src/main/cpp/util/_objs/md5/src/main/cpp/util/md5.pic.o): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
In file included from <built-in>:354:
<command line>:6:9: warning: '_FORTIFY_SOURCE' macro redefined [-Wmacro-redefined]
#define _FORTIFY_SOURCE 2
        ^
<command line>:2:9: note: previous definition is here
#define _FORTIFY_SOURCE 1
        ^
In file included from src/main/cpp/util/md5.cc:40:
./src/main/cpp/util/md5.h:21:10: fatal error: 'string' file not found
#include <string>
         ^
1 warning and 1 error generated.
Target //src:bazel failed to build
INFO: Elapsed time: 6.830s, Critical Path: 0.75s
builder for ‘/nix/store/r407rs601jvrz9ib7x55mi6cwkdyx0sv-bazel-0.3.1.drv’ failed with exit code 1
error: build of ‘/nix/store/r407rs601jvrz9ib7x55mi6cwkdyx0sv-bazel-0.3.1.drv’ failed
```